ip4defrag/defrag_test.go

```diff
@@ -237,7 +237,6 @@ func TestDefragMaxSize(t *testing.T) {
 
 	_, err := defrag.DefragIPv4(&ip2)
 	if err != nil {
-		t.Fatal(err)
 		t.Fatalf("defrag: Maximum length is supposed to be 65535")
```

Comment:
it would be good for debugging if the error were still printed, that can be done either by changing the removed Fatal to Error instead of removing or by adding the error to the next Fatalf.